bluebrain/repo-bluebrain/packages/neurodamus-model/package.py

```diff
@@ -31,10 +32,14 @@ def version_from_model_core_dep(model_v, core_v):
                when='@' + this_version)
 
 
-def version_from_model_ndpy_dep(model_v, ndamus_v=PYNEURODAMUS_DEFAULT_V):
-    """New version scheme following dependency on neurodamus-py
+def version_from_model_ndpy_dep(
+    model_v,
+    ndamus_v=PYNEURODAMUS_DEFAULT_V,
+    common_v=COMMON_DEFAULT_V
+):
+    """New version scheme following dependency on neurodamus-py and common
     """
-    this_version = model_v + "-" + ndamus_v  # e.g. 1.1-3.0.2
+    this_version = model_v + "-" + ndamus_v + "-" + common_v  # e.g. 1.1-3.0.2-2.6.4
```

Comment:
Since we're going to deploy anyway new modules with newer `py-neurodamus` version we can get away this time from having to add the `common` version suffix but I think this is something that we will come across in the future as well